util/util.go

```diff
@@ -38,6 +38,10 @@ func commonFlags() []cli.Flag {
 			Name:  "table",
 			Usage: "Return output in tabular format. This is the default output format.",
 		},
+		cli.BoolFlag{
+			Name:  "csv",
+			Usage: "Return output in csv format.",
+		},
```

Comment:
Anyone have thoughts on explicitly having each output format as its own flag vs. having a single format flag? Having each as its own flag ensures that users will get an error message if they provide an invalid format. However, having a single flag would cause less clutter (for example, during bash-completion).